src/javascript/SelectorTypes/Picker/PickerDialog/MainPanel.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import {withStyles, Button} from '@material-ui/core';
-import {buttonRenderer} from '@jahia/react-material';
+import {withStyles} from '@material-ui/core';
+import {Button} from '@jahia/moonstone';
+import {ButtonRenderer} from '~/actions/ActionsButtons';
 import {DisplayActions} from '@jahia/ui-extender';
+import {FileUpload} from 'mdi-material-ui';
```

Comment:
is it good, the import?

Reply:
I took the same as the one used before
https://github.com/Jahia/content-editor/blob/4c06205ac16b096bfe17f79f6428dc9df29dd4ff/src/javascript/SelectorTypes/Picker/actions/index.js#L2